keyboards/winkeyless/bface/matrix.c

```diff
@@ -99,8 +99,9 @@ uint8_t matrix_scan(void) {
             }
         }
     }
-
+#ifdef RGBLIGHT_ENABLE
     matrix_scan_user();
```

Comment:
This should really be calling `matrix_scan_quantum()` without the ifdef. When the user function in bface.c is not defined it defaults to an empty implementation, and gets compiled out.

In fact, bface.c should actually be implementing `matrix_scan_kb()`, and calling `matrix_scan_user()` after the RGB task. This way keymaps can still add their own code to the matrix scan event, without having to call `rgblight_task()` themselves.

Reply:
You're probably right, but that's outside the scope of this PR and my knowledge of QMK inner workings.

Reply:
Well, with the addition of this ifdef here, `matrix_scan_user()` will never get called if you have RGB disabled, which may break other people's keymaps.

Reply:
+1 for fixing it correctly. We shouldn't be breaking the existing behaviour by fixing something else.

If you have a preference, we can point to existing implementations to help show what needs to change, or prepare a PR against yours with the required changes.

Reply:
No preference, I'm happy either way if it gets fixed.

Reply:
If that's the case, could you at least remove the ifdef around the `matrix_scan_user` option?